packages/pipeline-editor/src/PipelineController/index.ts

```diff
@@ -133,6 +133,10 @@ class PipelineController extends CanvasController {
 
     // TODO: We should only set this for file based nodes.
     data.nodeTemplate.app_data.filename = item.path;
+    const properties = await item.onPropertiesUpdateRequested?.({
+      filename: item.path,
+    });
```

Comment:
With the coming changes where we fetch the palette from the server, we will need to request properties whenever the properties panel opens, so I think we should move this there as well maybe?

Reply:
I'm not sure where you think we should do that, but it sound like it could just be added once we make that change

Reply:
okay, Im fine with that